configs/infer_config.yaml

```diff
@@ -8,44 +8,69 @@
 # license agreement from NVIDIA CORPORATION is strictly prohibited.
 #
 
+### Data Pre-processing parameters
+## Generate intervals files
+intervalsize: 50000
+wg: False
+# Chromosome sizes to be used
+# You can also provide a file path
+genome: "hg19"
+# Ratio of non-peak to peak intervals in training
+# non-peak samples = nonpeak * peak samples
+nonpeak: "None"
+# If you do not wish to denoise whole genome,
+# You can specify a list of chromosomes
+# regions: ["chr1","chr2"]
+# You can also specify interval ranges for chromosomes
+# Equal sized intervals of intervalsize will be generated
+# within the mentioned intervals.
+#regions: ["chr1:0-10000","chr2:500-20000"]
+#You can also provide a path to a file containing
+#intervals that are equally spaced with intervalsize
+#Look at this file for reference:
+#https://github.com/clara-parabricks/AtacWorks/blob/master/tests/expected_results/end-to-end/result.holdout_intervals.bed
+regions: "None"
+
+## Input files
+noisybw: "None"
+cleanbw: "None"
+layersbw: "None"
+# Batch size to read the bigiwig files, because
```

Comment:
Some explanations of arguments are given in the config file while others are given in the help options in `cmd_args.py`. For consistency, should we keep all explanations in the help options alone and ask the user to use `-h` to understand the parameters?